pkg/flexvol/flexvol_windows.go

```diff
@@ -0,0 +1,103 @@
+/*
+(c) Copyright 2017 Hewlett Packard Enterprise Development LP
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package flexvol
+
+import (
+	log "github.com/hpe-storage/common-host-libs/logger"
+	"github.com/hpe-storage/common-host-libs/util"
+	"os"
+	"path"
+	"strings"
+)
+
+func doMount(flexvolPath, dockerPath, dockerName, mountID string) error {
+	// Get volume object for a disk mounted by plugin.
+	args := []string{"-command", "$vol = Get-Volume -FilePath", dockerPath, ";",
+		"Get-Partition -Volume $vol", "| Add-PartitionAccessPath -AccessPath", flexvolPath}
+	_, rc, err := util.ExecCommandOutput("powershell", args)
+	if rc != 0 || err != nil {
+		log.Tracef("Could not find mounted volume %s on the host, error %v", dockerPath, err.Error())
+		return err
+	}
+
+	return nil
+}
+
+// Unbind the mount and delete mountpoint
+func doUnMount(flexvolPath, mountID string) error {
+	log.Tracef("doUnmount called flexvolpath %s, mount id %s", flexvolPath, mountID)
+	//Get the docker volume name form the partition.
+	dockerVolumeName, err := getDockerVolName(flexvolPath)
+	log.Tracef("docker volume name %s", dockerVolumeName)
+	if err != nil {
+		log.Errorf("Could not find the docker volume name for flexvolPath %s  error %v", flexvolPath, err.Error())
+		return err
+	}
+	// Remove Partition Access Path
+	args := []string{"-command", "$vol = Get-Volume -FilePath", flexvolPath, ";",
```

Comment:
Same comment as above.  Let's move this to a windows package.

Reply:
Replied as above. thx